src/services/remote-config/remote-config-service.ts

```diff
@@ -24,7 +27,14 @@ export class RemoteConfig {
       if (RemoteConfig.isInited) {
         return RemoteConfig.getAll();
       }
-      const config = await Backend.instance.getRemoteConfig();
+      const uid = await getUid();
+      const wallets = Wallet.getAll().map(wallet => wallet.address);
```

Comment:
Could you please create new method which called `Wallet.getAllWallets: string[]`?